typed.go

```diff
@@ -67,17 +68,22 @@ func strToFelt(str string) *felt.Felt {
 	asciiRegexp := regexp.MustCompile(`^([[:graph:]]|[[:space:]]){1,31}$`)
```

Comment:
I wonder if we could replace strToFelt with utils.HexToFelt() (which doesn't actually require the strin to be a Hex, we should probably update the name)

Reply:
Don't think so, as they arent necessarily Hex values

Starknet.js has a similar function (https://github.com/0xs34n/starknet.js/blob/44d2706f9d862564f59328a2677b2359f9d3f3a6/src/utils/shortString.ts#L33)